framework/freedomotic-core/src/main/java/com/freedomotic/plugins/PluginIsolatedClassloader.java

```diff
@@ -53,20 +54,19 @@
      */
     public PluginIsolatedClassloader(String jarDir) throws FreedomoticException {
```

Comment:
This `throws FreedomoticException` should be removed since `FreedomoticRuntimeException` is unchecked.